### PR TITLE
docs: Update CHANGELOG.md for 0906 release

### PR DESCRIPTION
**Pull request by zyxiaoyuer**

### CODE REVIEW

**Review by wenytang-ms — APPROVED** [review]

File: packages/vscode-extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
 
+## 2.6.0 - Sep 06, 2021
+
+Incremental version for Teams Toolkit with multiple bugs fixed and the following updates：
+ 
+New Feature:
+- Support projects migration from Teams Toolkit V1 to V2. If your Teams projects are created using Teams Toolkit V1, try migrate your project follow the [migration instructions](https://aka.ms/teamsfx-migrate-v1).
+- Support local debug experience for Teams Tab/Bot/Message Extension project migrated from Teams Toolkit V1.
+- Check permission to turn-on Teams custom app uploading when user sign-in to M365 account. Learn more about [Teams app uploading or sideloading permission](https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant#enable-custom-teams-apps-and-turn-on-custom-app-uploading).
+- (Preview Feature) Support provision cloud resources using ARM template. Learn more about [ARM template](https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/overview). To enable this feature, please follow [instructions]( https://github.com/OfficeDev/TeamsFx/wiki/Enable-Preview-Features-in-Teams-Toolkit).
```
Comment (blackchoey) [inline packages/vscode-extension/CHANGELOG.md:11]: need to update the ARM template link to bicep one

**Review by zhenjiao-ms — CHANGES_REQUESTED** [review]
It seems 2 features are missing
- Simplified Bot
- Collaboration: List api and grand permission

**Comment by eriolchan** [comment]
> It seems 2 features are missing
> 
> * Simplified Bot
> * Collaboration: List api and grand permission

From @zyxiaoyuer's email, simplified bot is in "Enhancement" part, but it is missing in the final CHANGELOG.md. @zyxiaoyuer can you help check this?

**Comment by adashen** [comment]
> * i and grand permission

as for collaboration, it is currently hidden by feature flag so we didn't need to list in changelog.

**Comment by adashen** [comment]
> > * i and grand permission
> 
> @zhenjiao-ms as for collaboration, it is currently hidden by feature flag so we didn't need to list in changelog.

File: packages/vscode-extension/CHANGELOG.md (hunk 1 above)
Comment (zyxiaoyuer) [reply packages/vscode-extension/CHANGELOG.md:11]: could you please provide the link?

Comment (zyxiaoyuer) [reply packages/vscode-extension/CHANGELOG.md:11]: https://docs.microsoft.com/en-us/azure/azure-resource-manager/bicep/overview is this one you are referring?

**Comment by zyxiaoyuer** [comment]
> It seems 2 features are missing
> 
> * Simplified Bot
> * Collaboration: List api and grand permission

@zhenjiao-ms , I confirmed with feature PM in my email:
1. “remove lock file to system folder”, and “simplified bot folder structure” is included in feature point "- Optimization of created project folder structure." 
 
2. Collaboration is confirmed by Yanliang that will not go public but for internal preview.

File: packages/vscode-extension/CHANGELOG.md (hunk 1 above)
Comment (zyxiaoyuer) [reply packages/vscode-extension/CHANGELOG.md]: According to our offline discussion, remove the link to ARM doc.

**Review by zhenjiao-ms — APPROVED** [review]

**Pull request merged**